zerver/openapi/zulip.yaml

```diff
@@ -16320,6 +16321,12 @@ paths:
                                 modified by users.
 
                                 **Changes**: New in Zulip 5.0 (feature level 93).
+                            can_mention_group_id:
+                              type: integer
+                              description: |
+                                ID of the user group whose members are allowed to mention the group.
+
+                                **Changes**: New in Zulip 8.0 (feature level 190).
```

Comment:
We should spell out "Previously, groups could be mentioned if any only if they were not system groups." in the changes line.

Reply:
Added this line.